python_examples/examples/bulb_example.py

```diff
@@ -1,4 +1,5 @@
 from sinric import Sinric
+import logging
```

Comment:
Perhaps instead of the standard Python `logging` module, use `loguru` which makes logging a little bit more simple.

Reply:
Yes. I am still learning about loguru, I'll use it in future updates or it'll make my work easier if you can edit the code with loguru. :)

Reply:
Do not merge PR yet, Let me fork and make changes. I will add you as a reviewer.

Reply:
Okay.